rend3/src/renderer/mod.rs

```diff
@@ -410,6 +410,27 @@ impl Renderer {
         handle
     }
 
+    /// Duplicates an existing object in the renderer, returning the new
+    /// object's handle. The duplicated object will reference the same mesh,
+    /// material and transform as the original object.
```

Comment:
Looking good, one last nit and we should be good to go.

There should be a description of how the changes provided in `change` will override the existing handle's things. (The text as it is makes it seem change is ignored)